src/Components/Cards/VideoCard.jsx

```diff
@@ -1,39 +1,47 @@
 import { useState, useEffect } from "react";
-import { Link, useLocation } from "react-router-dom";
+import { Link, useLocation, useParams } from "react-router-dom";
 import { useAuth, useAxiosCalls, useModal, useVideo } from "../../Context";
 import { formatTimeDuration } from "../../Utils/formatTimeDuration";
 import Button from "../UI/Button/Button";
 import IconButton from "../UI/Button/IconButton";
+import axios from "axios";
 import "./Cards.css";
 
-export const VerticalCard = ({ videoDetail }) => {
+export const VideoCard = ({ videoDetail }) => {
   const {
     snippet: { title, publishedAt, thumbnails, channelTitle },
     contentDetails: { duration },
     statistics: { viewCount },
   } = videoDetail;
 
-  const {
-    auth: { token },
-  } = useAuth();
+  const { auth } = useAuth();
+  const { token } = auth;
 
   const {
     likeVideoOnServer,
     unLikeVideoOnServer,
     addToWatchlaterOnServer,
     removeFromWatchlaterOnServer,
     removeFromHistoryListOnServer,
+    deleteVideoFromPlaylistOnServer,
   } = useAxiosCalls();
 
-  const { setShowLogin } = useModal();
+  const { setShowLogin, setShowPlaylistModal } = useModal();
 
   const {
-    videoState: { watchlater, likes },
+    videoState: { watchlater, likes, singlePlaylist },
+    setTempVideo,
+    videoDispatch,
   } = useVideo();
 
+  const [trash, showTrash] = useState(true);
+  const { playlistId } = useParams();
+
   const { pathname } = useLocation();
 
   const onHistoryPage = pathname.includes("history") ? true : false;
+  const onHomePage = pathname.includes("videos") ? true : false;
```

Comment:
Ain't the ternary clause redundant? includes returns true or false, adding the ternary clauses is redundant here.

Reply:
Fixed in next branch.